Utilities/Maintenance/RestyleHttpToHttps.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+#==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          https://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#==========================================================================
+
+
+# Script to automate restyling in an ITK remote module.
+# `sed` is used to replace http with https links for
+# known valid cases.
+
+find . -type f | \
+    fgrep -v sed-commands | fgrep -v .git/ | fgrep -v ThirdParty | \
+    tr \\n \\0 | xargs -0 sed -i -r \
+        -e 's#http://a11yproject.com#https://a11yproject.com#g' \
```

Comment:
An additional note that I forgot: the code would also modify the `LICENSE` files, which should be avoided:
https://www.apache.org/licenses/LICENSE-2.0.txt

Reply:
Yes, as @Leengit noted we are expecting all license headers in ITK and remote modules to match the template in [https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h), to which a change from http -> https was recently accepted.

Would appreciate clarification on why this "should be avoided", the license at [https://www.apache.org/licenses/LICENSE-2.0.txt](https://www.apache.org/licenses/LICENSE-2.0.txt) references back to [http://www.apache.org/licenses/LICENSE-2.0.txt](http://www.apache.org/licenses/LICENSE-2.0.txt) but otherwise the difference seems trivial. We should be encouraging https links.

Reply:
I appreciate the other comments here, but I do not have the bandwidth to implement and test suggestions for getting all valid https links with curl testing. Someone else is welcome to either modify with a commit here or follow up with a separate PR.

Reply:
Thanks for clarifying @jhlegarreta. I see the distinction between the header notices and the `LICENSE` files, I disagree that there is cause for concern in modifications to [ITKHeader.h](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) and subsequent remote file headers as these are simply pointing to the Apache license. I could see more discussion around modifications to `LICENSE` files in remote modules as these are typically copy/pastes of the Apache license. At the same time, I would rather not invest the effort to apply changes in every single remote module again.  Who would be able to advise on the criticality of reverting http links in `LICENSE` files?

cc @dzenanz  @Leengit

Reply:
I am not sure we have an authoritative source. But @jcfr probably deals more with licenses than us, so he might provide useful advice.